source/_components/blackbird.markdown

```diff
@@ -39,17 +39,19 @@ host:
 zones:
   description: This is the list of zones available. Valid zones are 1, 2, 3, 4, 5, 6, 7, 8. Each zone must have a name assigned to it.
   required: true
-  type: integer
+  type: map
   keys:
-    name:
+    ZONE_NUMBER:
```

Comment:
should this be in capital letters?

Reply:
It should be replaced by a zone number... so yes?

Reply:
```suggestion
    zone_number:
```